dora/core/client/fs/src/test/java/alluxio/client/file/cache/LocalCacheFileInStreamTest.java

```diff
@@ -915,6 +929,17 @@ public FileInStream openFile(URIStatus status, OpenFilePOptions options)
       }
     }
 
+    @Override
+    public FileInStream openFile(UfsUrl ufsPath, OpenFilePOptions options)
```

Comment:
why this change?

Reply:
There are many openFile function in the LocalCacheFileInStreamTest.java, I just added a ufsurl version.